index.html

```diff
@@ -68,8 +68,8 @@ <h3>Monero is untraceable.</h3>
                             <img src="/img/fungible-monero.png" alt="Monero gold" class="main-icon">
                         </div>
                         <div class="col-lg-6 col-md-6 col-sm-8 col-xs-12 why-text">
-                            <h3>Monero is fungible.</h3>
-                            <p> Monero is fungible because it is private by default. In its current state, it is extremely unlikely that Monero will ever be blacklisted by exchanges or vendors due to its association in previous transactions.</p>
+                            <h3>Monero is fungible</h3>
+                            <p> Monero is <a href="https://getmonero.org/resources/moneropedia/fungibility.html"> fungible </a>because it is private by default. Units of Monero cannot be blacklisted by vendors or exchanges due to their association in previous transactions.</p>
```

Comment:
You can shorten the URL to `/resources/moneropedia/fungibility.html` so it works also in local development environment. Also usually the spaces are outside the `<a>`-tag so you don't introduce double spaces..